sidecar/internal/server/server.go

```diff
@@ -69,8 +76,17 @@ func (ss *SidecarServer) RegisterService(svc SidecarService) {
 // Init creates the internal gRPC server, and registers the SidecarServices.
 // and starts gRPC server.
 func (ss *SidecarServer) Start() {
-	// create the gRPC server and register services
-	ss.server = grpc.NewServer()
+	if ss.enableTLS {
+		creds, err := credentials.NewServerTLSFromFile("/etc/tls/tls.crt", "/etc/tls/tls.key")
```

Comment:
`"/etc/tls/tls.crt", "/etc/tls/tls.key"` Are we going to have these two files in all the pods in same location when TLS is enabled?

Reply:
Ideally yes. But I guess we can make it configurable later on if that is preferred.

Reply:
adding these as parameters